.agents/skills/nemoclaw-maintainer-classify-ci-failure/scripts/classify-ci-failure.mts

```
@@ -137,39 +209,52 @@ export default async function triage_nemoclaw_ci_failure(input: {
-      const content = await tools.read({ file_path: boundedPath, limit: 20000 });
+      const content = await runtime.read({ file_path: boundedPath, limit: 20000 });
       logLines = content.lines.map((line) => line.text);
       sourceTruncated ||= content.totalLines > content.lines.length;
     }
   } finally {
-    await run(`rm -rf -- ${q(logDir)}`, "Remove temporary CI log directory");
+    const cleanup = await run(`rm -rf -- ${q(logDir)}`, "Remove temporary CI log directory");
+    if (cleanup.exitCode !== 0)
+      throw await diagnosticError(
+        cleanup.stderr.text || cleanup.stdout.text || "Could not remove temporary CI log directory",
+      );
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)